lib/asciidoctor-pdf/converter.rb

```diff
@@ -153,6 +153,7 @@ def init_pdf doc
     register_fonts theme.font_catalog, (doc.attr 'scripts', 'latin'), (doc.attr 'pdf-fontsdir', ThemeLoader::FontsDir)
     @theme = theme
     @font_color = theme.base_font_color
+    @background_color = theme.page_background_color
```

Comment:
I don't think we need to make `@background_color` an instance variable. I'm only doing that for `@font_color` because it saves a lot of typing.